src/cleaning/acp.c

```diff
@@ -652,7 +652,6 @@ static void _acp_flush(struct acp_context *acp)
 		.cmpl_fn = _acp_flush_end,
 		.lock_cacheline = false,
 		.lock_metadata = true,
-		.do_sort = false,
 		.cmpl_queue = true,
 		.io_queue = cache->cleaner.io_queue,
 	};
```

Comment:
ocf_cleaner_sort_flush_data() before the ocf_cleaner_do_flush_data_async() here as well?

Reply:
And how about the one in ocf/src/mngt/ocf_mngt_flush.c:366 ?

Reply:
In ACP there is no need for sorting, because chunks operate on core lines, so they are sorted by definition.

Reply:
> And how about the one in ocf/src/mngt/ocf_mngt_flush.c:366 ?

In management flush all the flush containers are sorted by one huge global sort in src/mngt/ocf_mngt_flush.c:491.